docs/linter-config.md

```diff
@@ -0,0 +1,5 @@
+Recommended linter configuration is to use the `linter-eslint` and `atom-ide-ui` packages.  `linter-eslint` will prompt you to download `linter-ui-default` and other "dependencies" but that's not necessary if you're using `atom-ide-ui`.  These packages don't need any special configuration to work together - it should just happen automagically.
```

Comment:
Actually, it will [only prompt for `linter`](https://github.com/AtomLinter/linter-eslint/blob/1427e52364351cbbe5ca433c23c6e8a23f72bfd3/package.json#L141) (which _then_ will prompt for it's own dependencies such as `linter-ui-default`).